bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BookKeeperBuildersTest.java

```diff
@@ -28,12 +29,13 @@
 import java.util.HashMap;
 import java.util.Map;
 import org.apache.bookkeeper.client.BKException.BKClientClosedException;
-import org.apache.bookkeeper.client.BKException.BKDigestMatchException;
 import org.apache.bookkeeper.client.BKException.BKIncorrectParameterException;
 import org.apache.bookkeeper.client.BKException.BKNoSuchLedgerExistsException;
 import org.apache.bookkeeper.client.BookKeeper;
+import org.apache.bookkeeper.client.LedgerHandle;
 import org.apache.bookkeeper.client.LedgerMetadata;
 import org.apache.bookkeeper.client.MockBookKeeperTestCase;
+import static org.apache.bookkeeper.client.api.WriteFlag.DEFERRED_FORCE;
```

Comment:
put all `import static` together.

(again, how this is possible to pass checkstyle?)

Reply:
we do not have checkstyle in this package, sorry

Reply:
I found it is enabled here, no? why it doesn't work?
`<suppress checks=".*" files=".*[\\/]client[\\/](?:[^\\/]+$|(?!api)|(?!impl)[^\\/]+[\\/])"/>`

Reply:
I had disabled checkstyle locally :-(

Reply:
this enables checkstyle only for "impl" and "api"
anyway now checkstyle passes for me locally